### ISSUE TITLE
Mouse duplication on MacOS

### ISSUE THREAD

**Issue by Hovushka** [aseprite/aseprite#2135]
When switching between tools via shortcuts, cursor sometimes duplicates itself and its older instance remains on the screen. The clone disappears if you move your active cursor to another editor window or if you press the "command" key.

I am extremely sorry for the images bring provided as photos of the screen, but the bug was extremely hard to catch with the built-in system screenshot tool.

![IMG_6769](https://user-images.githubusercontent.com/43238639/63636328-7d2c3300-c676-11e9-9f23-d9f2dbddd9a5.jpg)

![IMG_6770](https://user-images.githubusercontent.com/43238639/63636332-87e6c800-c676-11e9-8924-0d8bc1b77c03.jpg)

This issue can be reproduced with both native and non-native cursor.

### Aseprite and System version

* Aseprite version: 1.2.13 portable
* System: macOS Mojave 10.14.6

**Comment by dacap** [comment]
Yes, I've noticed the same problem, not yet sure how to exactly reproduce it, but I think it was introduced in recent updates (Aseprite updates, or macOS updates). I'll try to fix it.

**Comment by Hovushka** [comment]
I've tested out the new version 1.2.14. I've managed to replicate the bug once, but all my other attempts were in vain.
